applications/advanced-concepts/multi-engine-machine-translation.md

```diff
@@ -0,0 +1,20 @@
+---
+description: Multi-engine machine translation
+---
+
+# Multi-engine machine translation
+
+**Multi-engine machine translation** is the process of evaluating and picking a machine translation engine. In multi-engine machine translation, different [providers](business/providers.md) translate the same input sample. The output versions are then evaluated automatically or by a human to select a provider.
+
+Multi-engine machine translation is challenging because it is subjective.
+
+### Goals
+
+The goal of multi-engine machine translation is to find the best fit for specific machine translation needs.
+The best fit for a particular input will vary depending on different factors:
+
+- Domains
```

Comment:
I think we should remove all of this because these are more project-level criteria, but the engine selection can be segment-level, and the benefit is really avoiding random catastrophic junk (i.e. limiting downside, not increasing quality in the ideal cases).